src/builtin/power/policy.rs

```diff
@@ -15,8 +15,14 @@ pub const CONSENSUS_MINER_MIN_MINERS: i64 = 3;
 /// To support onboarding 1EiB/year, we need to allow at least 32 prove commits per epoch.
 pub const MAX_MINER_PROVE_COMMITS_PER_EPOCH: u64 = 200;
 
+#[cfg(not(feature = "devnet"))]
 lazy_static! {
-    // TODO: Might need to be able to override later on based on network params
     /// Minimum power of an individual miner to meet the threshold for leader election.
     pub static ref CONSENSUS_MINER_MIN_POWER: StoragePower = StoragePower::from_i64(10 << 40).unwrap(); // placeholder
 }
+
+#[cfg(feature = "devnet")]
+lazy_static! {
+    /// Minimum power of an individual miner to meet the threshold for leader election.
+    pub static ref CONSENSUS_MINER_MIN_POWER: StoragePower = StoragePower::from_i64(2048).unwrap(); // 2k
+}
```

Comment:
You might be able to just put the feature tag on just the variable and not the lazy_static block, but doesn't matter

Reply:
Naw, doenst work... Probably has something to do with the fact that its in a macro.